.nvmrc

```diff
@@ -0,0 +1 @@
+6.10.0
```

Comment:
The `NodeJs` class uses this to determine Node version:

```ruby
          def use_nvm_default
            sh.if '-f .nvmrc' do
              sh.echo "Using nodejs version from .nvmrc", ansi: :yellow
              install_version '$(< .nvmrc)'
            end
            sh.else do
              install_version DEFAULT_VERSION
            end
          end
```

`DEFAULT_VERSION` is 0.10, which is too low a version for what we need, so unless I'm misunderstanding, adding `.nvmrc` should override that. 🙂